config/manager/kustomization.yaml

```diff
@@ -1,2 +1,8 @@
 resources:
 - manager.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: controller
+  newName: quay.io/brejohns0/etcd-operator/etcd-operator
+  newTag: v0.1
```

Comment:
can you explain why change it in this PR?

Reply:
It was a mistake, will be removed